pkg/driver/controller.go

```diff
@@ -709,6 +710,8 @@ func (d *ControllerService) CreateSnapshot(ctx context.Context, req *csi.CreateS
 		case FastSnapshotRestoreAvailabilityZones:
 			f := strings.ReplaceAll(value, " ", "")
 			fsrAvailabilityZones = strings.Split(f, ",")
+		case OutpostArnKey:
+			outpostArn = value
```

Comment:
Just a thought: should we validate that this is actually an ARN here? The request will fail either way when it reaches EC2, but we can avoid blasting unnecessary requests to EC2.

Reply:
In my opinion I don't see a situation where someone would use anything but their outpost arn as part of the VSC if they are using this param. Validating if this is actually an arn here to me feel like bloat but I am open minded and curious what others think.

Reply:
> I don't see a situation where someone would use anything but their outpost arn

This is hard to predict. :) 

Nevertheless, I'll defer this to the other team members.

Reply:
> Just a thought: should we validate that this is actually an ARN here?

+1, I think we should consider making use of validation that the SDK provides, see [func IsARN(arn string) bool](https://github.com/aws/aws-sdk-go-v2/blob/4ed838eab2a963cb16301501c8b8c3e29dac4c20/aws/arn/arn.go#L80).

Reply:
sounds good will do thank you guys !